R/plots.R

```diff
@@ -68,7 +77,8 @@ plot_individual <- function(data) {
         "<br>",
         "<b>Code:</b> ", code, "<br>",
         "<b>Description:</b> ", description, "<br>",
-        "<b>Code usage:</b> ", scales::comma(usage)
+        "<b>Code usage:</b> ", scales::comma(usage), "<br>",
+        "<b>Proportion of annual usage: </b>", annual_proportion, "%" 
```

Comment:
If you keep the decimals and dont multiply by 100 above we can use `scales::percent()` but feel free to ignore, not sure we winning anything by doing this

```suggestion
        "<b>Proportion of annual usage: </b>", scales::percent(annual_proportion, accuracy = 0.01)
```

Reply:
Agreed